build-data.stubs

```diff
@@ -24,6 +25,35 @@ use XML::XPath;
 my $xml = XML::XPath->new(filename => 'libphonenumber/resources/PhoneNumberMetadata.xml');
 my @territories = $xml->find('/phoneNumberMetadata/territories/territory')->get_nodelist();
 
+my %prefix_to_timezones;
+open(my $timezones_fh, "<:encoding(utf8)", 'libphonenumber/resources/timezones/map_data.txt')
```

Comment:
This file needs to be checked in [the relevant bit of build-data.sh](https://github.com/DrHyde/perl-modules-Number-Phone/blob/master/build-data.sh#L268) so that the script is fired off when the file is updated

Reply:
Done in https://github.com/DrHyde/perl-modules-Number-Phone/pull/134/commits/48627bc4ebfe71bbb52576e80500c28d5bc07a91